CONTRIBUTING.md

```diff
@@ -1,48 +1,53 @@
 # Contributing
- 
+
 Cool, that you are interested in contributing. We like pull request and suggestions from everyone.
 
 If you are planning in making a major contribution we appreciate your opening an issue or contacting [us](mailto:codecharta@github.com).
 
-Some things that will increase the chance that your pull request is accepted:
+Some things that will increase the chance of that your pull request to be accepted:
 
-Write clean code.
-Write tests.
-Write a good commit message.
-Code for Analysis sub-project only in Java and Kotlin languages.
+- Write clean code.
+- Write tests for your new code and regression tests after fixing a bug.
+- Write a good commit message and include the issue number at the end `#123`.
+- Code for Analysis sub-project only in Java and Kotlin languages.
+- Update the CHANGELOG.md with any changes/additions made
 
-We will *NOT* accept any pull requests that don't follow the Contributor Covenant [code of conduct](CODE_OF_CONDUCT.md).
+We will _NOT_ accept any pull requests that don't follow the Contributor Covenant [code of conduct](CODE_OF_CONDUCT.md).
 
 ## Naming conventions
 
 ### Branching
 
-Branch names consist of a type and the describing branch name itself, which is always **lowercase** separated by **dashes**. 
+Branch names consist of a type and the describing branch name itself, which is always **lowercase** separated by **dashes**.
 It follows this structure `<type>/<issue-id>/<name>`.
-For more detailed information check out the source [Branch Types by CKSource](https://docs.ckeditor.com/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html) 
+For more detailed information check out the source [Branch Types by CKSource](https://docs.ckeditor.com/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html)
 
-| Types         | Changelog     | Description
-| ---           | ---           | ---
-| `feature`	    | yes           | New feature I add or expand
-| `bugfix`	    | yes           | Bug fix
-| `codestyle`  	| no            | Our beloved code style improvements / refactorings
-| `docs`	    | no            | Updated documentation
-| `tech`	    | no            | Other kinds of technical changes
-| `revert`      | no            | Revert of some commit
+| Types       | Changelog | Description                                        |
+| ----------- | --------- | -------------------------------------------------- |
+| `feature`   | yes       | New feature I add or expand                        |
+| `bug`       | yes       | Bug fix                                            |
+| `docs`      | yes       | Updated documentation                              |
+| `codestyle` | no        | Our beloved code style improvements / refactorings |
+| `tech`      | no        | Other kinds of technical changes                   |
+| `revert`    | no        | Revert of some commit                              |
```

Comment:
```suggestion
| `revert`    | yes        | Revert of some commit                              |
```

I suggest to mention this as it could be substantial.